plugins/Terminal/js/components/commandinput.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { Map } from 'immutable'
+
+//These commands need a password prompt.
+const specialCommands = [ ['wallet', 'unlock'], ['wallet', 'load', 'seed'], ['help'], ['?'] ]
+
+const CommandInput = ({currentCommand, showCommandOverview, actions}) => {
+	componentDidUpdate: {
+		setTimeout(() => document.getElementById('command-input').focus(), 0)
+	}
+
+	reduce: {
```

Comment:
I'm confused by this syntax. Doesn't this need to be named `render`? And doesn't it need to be a function? This just looks like an object.